uaclient/entitlements/tests/test_esm.py

```diff
@@ -97,6 +155,17 @@ def fake_exists(path):
         assert add_apt_calls == m_add_apt.call_args_list
         assert 0 == m_add_pinning.call_count
         assert subp_calls == m_subp.call_args_list
+        if entitlement.name == "esm-infra":  # Then remove apt pref pin never
```

Comment:
A nit, but I initially read this as "Then remove apt pref pin, never". Perhaps just:

```suggestion
        if entitlement.name == "esm-infra":  # Remove "never" apt pref pin
```